src/utils/transformToHarRequest.js

```diff
@@ -0,0 +1,15 @@
+export function transformToHarRequest(data) {
```

Comment:
@MeganSteenkamp Thoughts on creating a generic "code-snippet" module that wraps the entire snippet generation pipeline (our converter AND the httpsnippet converter)?

Reply:
@tom-blake are you thinking that our `src/components/RequestCodePanel.astro` would call this module?

Reply:
I'm not sure because at the moment the generator pipeline processes all the files whereas in `RequestCodePanel.astro` we just want to call the httpsnippet converter for one snippet.

Reply:
I'm not sure because at the moment the converter runs for all files. This is kind of nice because we can clear out the dist directory. Whereas we only want to call `httpsnippet` for a specific file 🤔

Reply:
Outcome: This has been refactored and now we don't need any intermediate files YAY 🎉